### DISCUSSION TITLE
Setting up env file

### DISCUSSION THREAD

**Discussion by CodeforAbj** [cn10xdev/Event-Management#9] [Q&A] [answered]
How to provide this in env file
 senderEmail = 
 senderPass =
 sendgridKey =

**Comment by AkashSingh3031** [comment, marked as answer]
Hello @CodeforAbj,

1. First, create a new `.env` file in your forked repository and add the following values in the `.env` file:
   ```env
   SENDER_EMAIL=your-email-id
   SENDER_PASS=your-email-password
   SENDGRID_KEY=your-sendgrid-api-key
   ```

2. Next, create a `.gitignore` file (if it doesn't already exist) and add `.env` to it to ensure that the `.env` file is not pushed to GitHub:

   ```
   .env
   ```

This will prevent your sensitive information from being exposed in the live GitHub repository.
